media_receiver.cpp

```diff
@@ -26,8 +25,16 @@ namespace grt {
 			lock lck{ table_lck_ };
 			track_table_.emplace(info.id_, std::move(receiver));
 		}
-		
-								
+
+#if 0
+		auto recorder = set_video_recorder(
+			static_cast<webrtc::VideoTrackInterface*>(track.release()),
```

Comment:
this code may not work for both render and recording, if enabled.
to make it work.
we don't need to call track.release().